.github/workflows/pr-builder.yml

```diff
@@ -17,6 +17,9 @@ jobs:
   build:
     runs-on: ubuntu-latest
 
+    env:
+      JAVA_TOOL_OPTIONS: "-Djdk.util.zip.disableZip64ExtraFieldValidation=true -Djdk.nio.zipfs.allowDotZipEntry=true"
```

Comment:
why are we adding this?

Reply:
Added separately in https://github.com/wso2-extensions/identity-outbound-auth-sms-otp/pull/175